docker/builder/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM node:20-alpine
+RUN apk update && apk add --no-cache g++ make python3 && rm -rf /var/cache/apk/*
```

Comment:
What was the argument against using the `node:20` as builder image?

Reply:
I am really not sure it's worth to have a separate builder image just for these dependencies as the layer will always be retrieved from local cache after the first local build, and even if it is executed it just took 11 seconds on my machine.

The issues I see with using a separate builder image is that right now we automatically use the latest 20.x version which is good to get security updates in. There is also extra maintenance to keep this up-to-date if it's not part of the CI. Although we could build this image before running the main docker build.

Reply:
Moved back to a single `node` image.